Feature/73 expandable list data set

dkdhqt opened the pull request:
- 확장리스트 레이아웃, 데이터 적용
- 엔화원화 전환 처리

dkdhqt committed:
-확장리스트 데이터 적용
-엔화원화 전환 버튼 처리

dkdhqt committed:
-확장리스트 어댑터

dkdhqt committed:
-확장리스트 부모, 자식 레이아웃
-확장리스트 열기닫기 버튼 이미지

puze commented on app/src/main/java/com/jp_ais_training/keibo/adapter/ExpandableListAdapter.kt:
```diff
@@ -0,0 +1,93 @@
+    private val childList: MutableList<MutableList<CircleStatisticsFragment.MenuSpecific>>
+) : BaseExpandableListAdapter() {
+    companion object{
+        var setKRW = true
```
1. 클래스 이름은 ExpandableListAdapter이라 종속적이지 않고 공용으로 사용할 수 있는 클래스처럼 보이는데 setKRW라는 변수는 원그래프 화면에 종속적입니다.
클래스명을 바꾸거나 여기서 원그래프 화면에 종속적인 변수 혹은 메소드를 지우고 추상화와 캡슐화를 만족시켜야합니다.
2. boolean형 변수 명명법은 isKRW을 사용합니다.
3. static을 사용한 이유가 있다면 주석을 달고 특별한 경우가 아니면 객체지향에 반하기 때문에 지역변수로 선언해야합니다.

puze commented on app/src/main/java/com/jp_ais_training/keibo/frament/CircleStatisticsFragment.kt:
```diff
@@ -248,87 +300,86 @@ class CircleStatisticsFragment : Fragment() {
+        val arrMainName = getMainName.split(",")
+        val arrMainPrice = getMainPrice.split(",")
+
+        var k = 0
```
for (int i =0; i < arrMainPrice.size -1; i++) ??

dkdhqt committed:
데이터 없을 경우 표시할 이미지

dkdhqt committed:
원그래프 화면 확장리스트 어댑터
- 파일명 수정
- 기존 환율처리 기능 원그래프프래그먼트로 이동

dkdhqt committed:
원그래프 항목용 색상

dkdhqt committed:
원그래프 화면 레이아웃
- 데이터 없을 경우 표시할 이미지, 텍스트 추가

dkdhqt committed:
원그래프 화면 프래그먼트
- childValueList 변수 private 추가
- 원그래프 항목 색상 지정
- 환율 처리 기능 추가 (원화로 표시할 경우 환율 곱하기)
- 환율 처리 변수 int에서 boolean으로 변경
- 원그래프 항목 텍스트 크기 수정
- 데이터 유무에 따른 화면 처리 기능 추가
